### PR TITLE
Feature/apple touch icon

### PR DESCRIPTION
**Pull request by NanaKay007**
@phirework  This is a fix for the issue mentioned in #2718 
I tested this on my iphone 8+ device (via remote debugging), and it works.

### CODE REVIEW

File: web/index_template.html [hunk 1]
```diff
@@ -51,6 +51,7 @@
       href="https://fonts.googleapis.com/css?family=Zilla+Slab:400,600"
       rel="stylesheet"
     />
+    <link rel="apple-touch-icon" href="./apple-touch-icon-150x150.png" />
```
Comment (phirework) [inline web/index_template.html:54]: Can you use the same convention as the other static icon images in this file, i.e. using the static img file? Also, I don't think the filename itself has to be called apple-touch-icon or whatever, as long as it's specified in the `rel`. 

I've gotten a better icon for this from the design folks as well. 
CV iOS App Icon

Comment (NanaKay007) [reply web/index_template.html:54]: Hmmm...I dont know what you mean by 'static img file'. Is this what you mean?

`<link rel="apple-touch-icon" href="https://voice.mozilla.org/img/voice-touch-icon.png" />`

Comment (phirework) [reply web/index_template.html:54]: Yep exactly. We're using the `/img` folder for static meta image assets, so let's stick to the convention and avoid unnecessary files on the root level.

**Comment by NanaKay007** [comment]
forgot to do that. thanks

**Pull request merged**